src/base/preferences.cpp

```diff
@@ -92,17 +93,41 @@ void Preferences::setValue(const QString &key, const QVariant &value)
     SettingsStorage::instance()->storeValue(key, value);
 }
 
-// General options
+// Appearance/Language options
 QString Preferences::getLocale() const
 {
-    return value("Preferences/General/Locale", QLocale::system().name()).toString();
+    return value("Preferences/Appearance/Locale", QLocale::system().name()).toString();
 }
 
 void Preferences::setLocale(const QString &locale)
 {
-    setValue("Preferences/General/Locale", locale);
+    setValue("Preferences/Appearance/Locale", locale);
 }
 
+#if (defined(Q_OS_UNIX) && !defined(Q_OS_MAC))
+bool Preferences::useSystemIconTheme() const
+{
+    return value("Preferences/Appearance/useSystemIconTheme", true).toBool();
```

Comment:
Why we need "Preferences" in these paths? Why not just "Appearance/useSystemIconTheme" and so on?